src/Io/IniUtil.php

```diff
@@ -31,18 +31,18 @@ public static function iniSizeToBytes($size)
         }
 
         if ($suffix === 'K') {
-            return $strippedSize * 1024;
+            return (int)($strippedSize * 1024);
```

Comment:
the previous could potentially return a float, not an int

Reply:
Note that an integer overflow will automatically cast to float in PHP. This means that `8G` on a 32 bit system will return the correct float value, while your change enforces an int and thus limits this value to` PHP_INT_MAX` again. I'm not sure how relevant this is here, but certainly something we should be aware of.

How about using a `number` type hint instead here? http://php.net/manual/de/language.pseudo-types.php#language.types.number